distributor-api-v1/deprecations/payment-card-input.md

```diff
@@ -0,0 +1,154 @@
+# PaymentCardInput deprecation
+[PaymentCardInput](#paymentcardinput) field of [configuration](../operations.md#configuration) object is deprecated and will be removed from the response of [configuration/get](../operations.md#getconfigurationinfo) on 31.7.2021. It is replaced by [PaymentCardRequirement](#paymentcardrequirement). The new field contains two new options for requiring a payment card.
+
+## Response diff
+
+```diff
+{
```

Comment:
Is it worth having it? It's really long. If the change in the table refers to the object, the user can see the diff. Depends on you.